helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -167,6 +173,14 @@
   private static final int GLOBAL_TARGET_TASK_THREAD_POOL_SIZE_NOT_SET = -1;
   private static final int OFFLINE_NODE_TIME_OUT_FOR_MAINTENANCE_MODE_NOT_SET = -1;
 
+  private static final Map<String, String> DEFAULT_SUPER_CLUSTER_REBALANCE_CONFIG = ImmutableMap
+      .of(RebalanceConfig.RebalanceConfigProperty.REBALANCE_MODE.name(),
+          RebalanceConfig.RebalanceMode.FULL_AUTO.name(),
+          RebalanceConfig.RebalanceConfigProperty.REBALANCER_CLASS_NAME.name(),
+          DelayedAutoRebalancer.class.getName(),
```

Comment:
I thought it's WAGED by default?

Reply:
We cannot change the existing default behavior. So this PR is to add the configuration item to make WAGED default.

Reply:
@NealSun96 as discussed with Junkai, we decided to make WAGED the default option : )